src/Core/Resolvers/CosmosQueryEngine.cs

```diff
@@ -311,7 +311,7 @@ private async Task<(string? idValue, string? partitionKeyValue)> GetIdAndPartiti
         /// <param name="parameter"></param>
         /// <returns></returns>
 #nullable enable
-        private string? GetPartitionKeyValue(string? partitionKeyPath, object? parameter)
+        private static string? GetPartitionKeyValue(string? partitionKeyPath, object? parameter)
```

Comment:
why is this change needed in this PR?

Reply:
because this method was not using any instance data, so compiler was giving warnings.